libexec/cli/build.help

```diff
@@ -118,6 +118,22 @@ DEFFILE SECTION EXAMPLES:
         export HAN VADER LUKE
 
 
+DEFFILE SCI-F SECTION EXAMPLES:
+
+    %appinstall app1
+        echo "These are steps to install an app using the SCI-F format"
+
+    %appenv app1
+        APP1VAR=app1value
+        export APP1VAR
+
+    %apphelp app1
+        This is a help doc for running app1
+
```

Comment:
for this an the above, if you are doing a full example, might make sense to include `%applabels` and `%appfiles`. Also included (and not likely to be used) are for setup and test too. Maybe we can put the major ones here, and link to your cowsay example (with the more "advanced" usage there?

Reply:
Yes.  Can you point me to a place where I can see an example of these sections too in the docs so that I can add to the help?